pkg/bff/docs/swagger.yaml

```diff
@@ -652,6 +652,34 @@ paths:
       summary: Create a new organization [create:organizations]
       tags:
       - organizations
+  /organizations/{id}:
+    delete:
+      description: Completely delete an organization, including the registration and
```

Comment:
Should we add a little note like "Warning! Deletes are not reversible! Be sure to prompt user for confirmation before executing."